src/WebJobs.Script/Description/DotNet/ScriptFunctionMetadataResolver.cs

```diff
@@ -55,7 +55,7 @@ public sealed class ScriptFunctionMetadataResolver : MetadataReferenceResolver,
         private static readonly List<ISharedAssemblyProvider> SharedAssemblyProviders = new List<ISharedAssemblyProvider>
             {
                 new DirectSharedAssemblyProvider(typeof(Newtonsoft.Json.JsonConvert).Assembly), /* Newtonsoft.Json */
-                new DirectSharedAssemblyProvider(typeof(WindowsAzure.Storage.StorageUri).Assembly), /* Microsoft.WindowsAzure.Storage */
+                new DirectSharedAssemblyProvider(typeof(Microsoft.WindowsAzure.Storage.StorageUri).Assembly), /* Microsoft.WindowsAzure.Storage */
```

Comment:
Do we want to add the new assemblies to this as well? I can't remember if we discussed it. Maybe it's best to just not do this? @fabiocav may have an opinion.

Reply:
So I omitted adding it here, because it was not my intention to allow users to #r Microsoft.Azure.* (unless they add a reference themselves) however see my other comment below...